test/e2e/set/scyllacluster/scyllacluster_upgrades.go

```diff
@@ -5,7 +5,6 @@ package scyllacluster
 import (
 	"context"
 	"fmt"
-
```

Comment:
Why remove this whitespace? Do we have a goimports-like standard for the import block structure?

Reply:
fixed, my IDE was missing import reformat action on save.